testnet/stacks-node/Cargo.toml

```diff
@@ -8,18 +8,26 @@ edition = "2018"
 lazy_static = "1.4.0"
 pico-args = "0.3.1"
 rand = "=0.7.2"
-reqwest = { version = "0.10", features = ["blocking", "json", "native-tls-vendored"] }
 secp256k1 = { version = "0.11.5" }
 serde = "1"
 serde_derive = "1"
 serde_json = { version = "1.0", features = ["arbitrary_precision", "raw_value"] }
 stacks = { package = "blockstack-core", path = "../../." }
 toml = "0.5.6"
+async-h1 = "1.1"
+async-std = { version = "<1.6", features = ["attributes"] }
+http-types = "1.0"
+base64 = "0.12.0"
 
 [dev-dependencies]
 warp = "0.2"
 tokio = "0.2.21"
+reqwest = { version = "0.10", features = ["blocking", "json", "native-tls-vendored"] }
```

Comment:
Can you change this to use `rusttls`?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1670/commits/be246a023a00a7b1f54b393c59a2436d7ec77685